web/Dockerfile

```diff
@@ -1,11 +1,23 @@
-FROM nginx:mainline-alpine
+FROM nginx
```

Comment:
Any chance make it work with Alpine image?

Reply:
I can try to make work on alpine. Let me take a look on that.